src/pages/token/StakingRewardPanel.tsx

```diff
@@ -34,7 +34,7 @@ export const StakingRewardPanel = ({ usdcRewards }: ElementProps) => {
   const stringGetter = useStringGetter();
 
   const canAccountTrade = useAppSelector(calculateCanAccountTrade);
-  const chartDotsBackground = useAppSelector(getChartDotBackground);
+  const chartDotsBackground = useAppSelector(getBackground)(BackgroundType.Dots);
```

Comment:
[suggestion] feel like this would be easier to read if we had 2 selectors `getChartDotBackground` and `getChartGridBackground`. it would also allow us to localize the changes to just the file that we want to update (less vectors for regressions)